testing/emulationstation/APKBUILD

```diff
@@ -0,0 +1,63 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=emulationstation
+pkgver=2.7.1
+pkgrel=0
+pkgdesc="EmulationStation is a cross-platform graphical front-end for emulators with controller navigation."
+url="https://github.com/RetroPie/EmulationStation"
+arch="all"
+license="MIT"
+depends="$pkgname-theme-default"
+makedepends="boost-dev freeimage-dev freetype-dev sdl2-dev mesa-dev alsa-lib-dev vlc-dev curl-dev cmake"
+subpackages="$pkgname-theme-default:theme:noarch"
+options="!check"
+_distname="EmulationStation"
+_pugixml_git="https://github.com/zeux/pugixml.git"
+
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/RetroPie/${_distname}/archive/v${pkgver}.tar.gz
```

Comment:
Wrapping the variables in {} is not needed in most cases.

Reply:
Done.